### PR TITLE
Added warnings to the medbeam gun re: stream-crossing

### PR DESCRIPTION
**Pull request by IAmBigCoat**
**What does this PR do:**
Adds several warnings to the medbeam gun in its description, both on examine and in the uplink.
Also throws a message to everyone who can see medbeams cross and explode when the aforementioned medbeams cross and explode,
Now you can yell at people to read the warning on the gun!

**Images of sprite/map changes (IF APPLICABLE):**
image

**Changelog:**
:cl:
tweak: Added explosion warnings to medbeams, because medbeams can cause explosions. DON'T CROSS THE BEAMS!
/:cl:

### CODE REVIEW

**Comment by Citinited** [comment]
I thought a medbeam gun could have only one beam, whereas these added messages imply that a medbeam gun can have more than one beam. Could they be reworded?

**Comment by davidchan13** [comment]
One beam per gun. But nukes and oddys can get them, so if they criss cross targets they go boom.

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -1351,7 +1351,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 
 /datum/uplink_item/device_tools/medgun
 	name = "Medbeam Gun"
-	desc = "Medical Beam Gun, useful in prolonged firefights."
+	desc = "Medical Beam Gun, useful in prolonged firefights. DO NOT CROSS THE BEAMS. Crossing the beams or attaching two beams to one target will have explosive consequences."
```
Comment (Ralta) [inline code/datums/uplink_item.dm]: I'd recommend rewording slightly to indicate this only occurs when another medbeam gun is used:
`"DO NOT CROSS THE BEAMS. Crossing beams with another Medbeam Gun or attaching two beams to one target will have explosive consequences."`

**Comment by MrMagolor** [comment]
>One beam per gun. But nukes and oddys can get them, so if they criss cross targets they go boom

Wait, there's an exosuit medibeam gun? How do you get/make it?

**Comment by KasparoVy** [comment]
# DO _KNOW_ HARM

File: code/datums/uplink_item.dm (hunk 1 above)
Comment (IAmBigCoat) [reply code/datums/uplink_item.dm]: IT IS DONE.

**Pull request merged**